.travis.yml

```diff
@@ -14,53 +14,108 @@
 
 language: c
 
-compiler:
-  - gcc
+compiler: gcc
 
-before_script:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y valgrind
+jobs:
+  include:
 
-  - git submodule update --init --recursive
-  - mkdir build
-  - cd build
+  # Release build, gcc
+  - if: not branch = coverity_scan
```

Comment:
Maybe let all these checks run on the `coverity_scan` branch too, just keep the `.travis.yml` more simple.  All the `if not`s distract from the common case.  It doesn't hurt anything to run all the checks - just a few a extra minutes of build time.

Reply:
👍